src/symtab/symbol_properties.hpp

```diff
@@ -180,9 +180,6 @@ enum class NmodlType : enum_type {
     /// constant variable
     constant_var = 1L << 21,
 
-    /// Partial Block
-    partial_block = 1L << 22,
```

Comment:
@alkino: as we removed 22, maybe we should shift the next symbols (i.e. avoid gap)? I will push the small change.